pytorch_toolkit/ote/requirements.txt

```diff
@@ -1,2 +1,5 @@
 pyyaml
 mmcv
+requests
+texttable
+torch
```

Comment:
Why `torch`?

Reply:
I wonder (I _really_ wonder), but `mmcv.Config` does not work without `torch`!
(You can see in `mmcv` package the files `mmcv/__init__.py` and `mmcv/utils/__init__.py`)

I could import the class `mmcv.Config` as `mmcv.utils.config.Config` -- that will work, but as far as I remember there is a general rule:
_if a package contains `__init__.py` you should not import classes  by other ways except `__init__.py`_
So I decided to add `torch` to requirements.txt

Reply:
The `ote` is designed as an auxiliary package with common functionality but now it overgrows with dispensable dependencies. It's sad... In light of this the `mmcv.utils.config.Config` import is not so bad, IMHO.

Reply:
I agree that it is sad to add to ote such big dependencies as `torch`.
Especially if we think that different domains may require different version of pytorch.
But it is not so bad: any `init_venv.sh` in the folder `pytorch_toolit` installs some version of pytorch, and `ote` package is installed in `init_venv.sh` after all packages, and any version of `torch` is sufficient for `ote`, so this dependency should not break anything.

But I agree that it is counterintuitive to use `torch` just to convert config files.

So, we can use `mmcv.utils.config.Config`.
The only drawback of this decision: if in the future `mmcv` authors change something in the structure of `mmcv` package we will have to make more changes.

So, I wait opinion of @Ilya-Krylov : whatever he decides.

Reply:
no torch